components/ui/theme-toggle.tsx

```
@@ -0,0 +1,43 @@
+
+import { Button } from "@/components/ui/button";
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+} from "@/components/ui/dropdown-menu";
+import { useTheme } from "@/hooks/use-theme";
+
+export function ThemeToggle() {
+  const { setTheme, resolvedTheme } = useTheme();
```

ESLint

[error] 'resolvedTheme' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)